docs/writing-tests/integrations/vitest-addon/migration-guide.mdx

```diff
@@ -0,0 +1,143 @@
+
+### 1. Remove test-runner dependencies
+
+Remove the following dependencies:
+
+- `@storybook/test-runner` dependency
+- `@storybook/addon-coverage` dependency (if present)
+
+### 2. Remove test-runner related files
+
+Remove the following files:
+
+- `test-runner-jest.config.js` file (if present)
+- `.storybook/test-runner.ts` file (if present)
+  - If you had configured accessibility testing, this is now done out of the box with the Vitest addon.
+  - If you had configured image snapshot testing, check [this FAQ](#what-if-i-am-doing-image-snapshot-testing).
+  - If you had configuration for custom, advanced use cases such as using [the `prepare` hook](https://github.com/storybookjs/test-runner#prepare), please reach out on [Discord](https://discord.gg/storybook) or open a [discussion on GitHub](https://github.com/storybookjs/storybook/discussions/new/choose) so we can assist you with the migration.
+
+### 3. Set up `@storybook/addon-vitest`
+
+- Run this command, which will set up all the necessary code for you.
+  
+  <CodeSnippets path="addon-test-install.md" />
+  
+- Update the `package.json` script for Storybook tests, replacing the test-runner binary and using Vitest instead:
+
+  ```diff
+  {
+    "scripts": {
+  -    "test-storybook": "test-storybook"
+  +    "test-storybook": "vitest --project=storybook"
+    }
+  }
+  ```
+
+- Visit [the Vitest addon guide](./index.mdx) to learn how to use it, how to configure it, and how it works.
+
+Tips:
+
+1. If you previously used `--coverage` with the test-runner, use Vitest’s coverage instead: `vitest --project=storybook --coverage`.
+2. Add more browsers by extending [`instances` field](https://vitest.dev/guide/browser/config.html#browser-instances) in the Vitest configuration file (e.g., `firefox`, `webkit`).
+
+### 4. Update your CI to run Vitest instead of the test-runner
+
+If you're using the Storybook test-runner in CI, replace its build/serve/wait flow with a single Vitest command. For example, in GitHub Actions:
+
+```diff
+# .github/workflows/storybook-tests.yml
+name: 'Storybook Tests'
+
+on: push
+
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+      - name: Install dependencies
+        run: yarn
+      - name: Install Playwright
+        run: npx playwright install --with-deps
+-      - name: Build Storybook
+-        run: yarn build-storybook --quiet
+-      - name: Serve Storybook and run tests
+-        run: |
+-          npx concurrently -k -s first -n "SB,TEST" -c "magenta,blue" \
+-            "npx http-server storybook-static --port 6006 --silent" \
+-            "npx wait-on tcp:127.0.0.1:6006 && yarn test-storybook --coverage"
++      - name: Run storybook tests
++        run: |
++          npx vitest --project=storybook
+```
+
+If you want coverage in CI, run `vitest --project=storybook --coverage`.
```

Comment:
I'll leave it to you, but I would remove this sentence, as we're duplicating the same information that's already mentioned above in the tips section, and leave only the CI-related documentation

Reply:
It's just a single sentence, and I think it's a useful tip.